app/Repositories/Eloquents/WebhookRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace App\Repositories\Eloquents;
+ 
+use App\Models\Webhook;
+use App\Repositories\Interfaces\WebhookRepositoryInterface;
+use Auth;
+ 
+class WebhookRepository implements WebhookRepositoryInterface {
+	
+	private $model;
+
+    public function __construct(Webhook $model)
+    {
+        $this->model = $model;
+    }
+
+    public function getAllByUser()
+    {
+        return Auth::user()->webhooks()
+                                ->orderBy('webhooks.status', 'desc')
+                                ->orderBy('webhooks.created_at', 'desc')
+                                ->get();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/WebhookRepository.php</br>**Line**: 24</br>Expected 1 newline at end of file; 0 found